docs/Reek-Driven-Development.md

```diff
@@ -14,26 +16,31 @@ Reek::Rake::Task.new do |t|
 end
 ```
 
-Now the command `reek` will run Reek on your source code (and in this case, it fails if it finds any smells). For more detailed information about Reek's integration with Rake, see [Rake Task](Rake-Task.md).
+Now, `rake reek` will run Reek on your source code. Aand, in this case, it fails if it finds any smells.
+
+For more detailed information about Reek's integration with Rake, see [Rake Task].
+
+[Rake Task]: Rake-Task.md
```

Comment:
I'm confused about why you put this on a separate line. Shouldn't this be at the end of the line above?

Reply:
Oh, this is a Markdown "reference" link.

Reply:
https://daringfireball.net/projects/markdown/syntax#link - there are a few styles listed there, and GitHub-flavored Markdown supports even more.

Reply:
Ah, didn't know this 😆 TIL, I guess :)